product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/connecting_from_gcp/index.mdx

```diff
@@ -57,76 +57,75 @@ To walk through an example in your own environment, you need a:
 - Subnet in the VM’s VPC in the same region as the BigAnimal cluster.
 
 
-### Step 1: Publish a service from BigAnimal
+#### Step 1: Publish a service from BigAnimal
 
 !!! Note
-    Perform this procedure in the Google Cloud project connected to your BigAnimal subscription.
+    Publish a service from BigAnimal in the Google Cloud project connected to your BigAnimal subscription.
 
-In the Google Cloud project connected to BigAnimal, create a PSC Published Service to provide access to your cluster from other VPCs in other Google Cloud projects. Perform this procedure for each Postgres cluster to which you want to provide access.
+In the Google Cloud project connected to BigAnimal, to provide access to your cluster from other VPCs in other Google Cloud projects, create a PSC published service. Publish a service from BigAnimal for each Postgres cluster to which you want to provide access.
 
-1. Get the hostname of your Postgres cluster from the Connect tab of the Cluster page on the BigAnimal portal (`P-mckwlbakq5.private.brcxzr08qr7rbei1.biganimal.io`).
+1. Get the hostname of your Postgres cluster from the **Connect** tab of the Cluster page on the BigAnimal portal (`P-mckwlbakq5.private.brcxzr08qr7rbei1.biganimal.io`).
 
-1. Using Cloudshell, the command prompt, or other terminal, get the internal IP address of the host by performing a ping, nslookup, or dig +short &lt;host> against the hostname (`10.247.200.9`).
+1. Using Cloudshell, the command prompt, or some other terminal, get the internal IP address of the host by performing a ping, nslookup, or dig +short &lt;host> against the hostname (`10.247.200.9`).
 
 1. In the Google Cloud portal, go to **Network Services > Load balancing**.
 
-1. In the Filter area, choose Addresses under **LOAD BALANCERS**, and filter for the host IP (`10.247.200.9`). Note the load balancer name (`a58262cd80b234a3aa917b719e69843f`).
+1. In the Filter area, under **Load Balancers**, select **Addresses** and filter for the host IP (`10.247.200.9`). Note the load balancer name (`a58262cd80b234a3aa917b719e69843f`).
```

Comment:
I added links in the PR description to the source material. One of them has screenshots. I made some judgement calls based on the screenshots. Do you mind checking those screenshots and making sure that we are bolding the right things?